Python/load_keywords_and_URL_list.py

```diff
@@ -2,4 +2,19 @@
 This file is to contain a function for loading the source URL list and search keywords from a file.
 input:          filename
 output:         data structures containing URL list and search keywords list
+'''
+
+
```

Comment:
Need error handling for:
  file doesn't exist
  no file name (add to unit test)
  wrong file extension (only .txt's)